.github/workflows/pythonpublish.yml

```diff
@@ -57,10 +57,11 @@ jobs:
             echo "No tagged version found, exiting"
             exit 1
           fi
-          LINK="https://pypi.org/project/flytekitplugins-pod/${VERSION}"
+          LINK="https://pypi.org/project/flytekitplugins-pod/${VERSION}/"
           for i in {1..60}; do
-            if curl -L -I -s -f ${LINK} >/dev/null; then
-              echo "Found pypi"
+            result=$(curl -L -I -s -f ${LINK})
```

Comment:
What's the difference between these two?

Reply:
nothing - but it prints the link next.  and explicitly check $? - not sure if that's what the if was doing before.

Reply:
This was doing this already, but this code you wrote is more clear to the uninitiated. I like it.